Gopkg.toml

```diff
@@ -60,6 +60,10 @@ required = [
   name = "github.com/blang/semver"
   version = "v3.6.1"
 
+[[override]]
+  name = "github.com/coreos/prometheus-operator"
+  version = "v0.29.0"
```

Comment:
This is not needed here, but is included here for the future work that uses operator-sdk metrics.